feconf.py

```diff
@@ -617,6 +617,15 @@ def get_empty_ratings():
 # the existing storage models for UserStatsModel.
 DASHBOARD_STATS_DATETIME_STRING_FORMAT = '%Y-%m-%d'
 
+# Timestamp in sec since epoch for Mar 1 2021 12:00:00 UTC for the earliest
+# datetime that a report could be received.
+EARLIEST_APP_FEEDBACK_REPORT_DATETIME = datetime.datetime.fromtimestamp(
+    1614556800)
+
+# The minimum and maximum package version codes for Oppia Android.
+MINIMUM_ANDROID_PACKAGE_VERSION_CODE = 1
+MAXIMUM_ANDROID_PACKAGE_VERSION_CODE = 1
```

Comment:
Suppose the Android package version is updated in the future -- what is the process for updating this in the Web codebase so that no error arises, given that the web codebase only does monthly releases which are sometimes delayed? /cc @BenHenning 

(I'm wondering if we should even have a maximum version.)

Reply:
Ah good point. I'm not sure if there's a way to account for the asynchronous Android updates so I'd agree to not have a maximum version. Before making the change, WDYT Ben?

Reply:
Flagging as still unresolved. @BenHenning -- any thoughts?

Reply:
After thinking a bit more, I'm more inclined to remove this maximum value since there isn't a reliable way to every check this if releases will continue to be async. Seems silly to add any sort of "buffer" since that defeats the point of the check. I believe we also already check for positive integer values for the version code, which may be enough validation here. Haven't made this change yet but LMK any thoughts.

Reply:
I think I agree with your reasoning, yeah.

Reply:
I've made this change in the PR now.

Reply:
I agree with removing the maximum. Android & Play Store are designed to support very large numbers of APK versions, and we'll likely introduce automatic API generation in the future where we increment & throw away version codes (this is partly what they're meant to be used for).

Reply:
Ack -- I think we're running into the same problem in the request headers now, though ([this thread](https://github.com/oppia/oppia/pull/12367#discussion_r682925300)).